### PR TITLE
[Feat][Mobile] Pending campaigns API

### PR DESCRIPTION
**Pull request by kreazyme**
## Description

Pending campaigns API 🖊️ 

## Type of Change

- [x] ✨ New feature (non-breaking change which adds functionality)

### CODE REVIEW

File: mobile/assets/locales/en.json [hunk 1]
```diff
@@ -128,9 +128,9 @@
     "management": "Campaign Management",
     "empty_list": "Your organization doesn't have any campaigns, let's try creating new one!",
     "see_more": "See more",
-    "donator": "Donator",
+    "donator": "Donor",
```
Comment (dungngminh) [inline mobile/assets/locales/en.json:131]: Please update key lol :))))

File: mobile/assets/locales/en.json [hunk 2]
```diff
@@ -128,9 +128,9 @@
...
     "update_to": "Update to",
-    "list_donator": "List of donators",
+    "list_donator": "List of donors",
```
Comment (dungngminh) [inline mobile/assets/locales/en.json:133]: same as above

File: mobile/lib/modules/profile/widgets/profile/list_status_shimmer.widget.dart [hunk 3]
```diff
@@ -16,62 +18,66 @@ class ListStatus extends StatelessWidget {
...
-                      width: 30,
-                      height: 35,
-                      decoration: BoxDecoration(
-                        color: Colors.white,
-                        borderRadius: BorderRadius.circular(5),
-                      ),
+    return BlocBuilder<AuthBloc, AuthState>(
+      builder: (context, state) {
+        return isLoading
+            ? Row(
```
Comment (dungngminh) [inline mobile/lib/modules/profile/widgets/profile/list_status_shimmer.widget.dart]: If widget is too long, I think you should extract to the method like `_buildShimmerLoadingWidget()` or something else for readability.

Comment (kreazyme) [reply mobile/lib/modules/profile/widgets/profile/list_status_shimmer.widget.dart]: 👀

**Pull request merged**